src/Apps/Feature/Components/FeatureFeaturedLink.tsx

```diff
@@ -40,37 +36,33 @@ const MetaCell = styled(Flex)`
 
 export interface FeatureFeaturedLinkProps extends SpaceProps {
   wide?: boolean
-  featuredLink: {
-    title: string
-    href: string
-    subtitle?: string
-    description?: string
-    image: {
-      width: number
-      height: number
-      url: string
-    }
-  }
+  featuredLink: FeatureFeaturedLink_featuredLink
 }
 
 export const FeatureFeaturedLink: React.FC<FeatureFeaturedLinkProps> = ({
   wide,
-  featuredLink: { title, href, subtitle, description, image },
+  featuredLink,
+  featuredLink: { href, title, subtitle, description, image },
   ...rest
 }) => {
   return (
     <Container to={href} {...rest}>
       {image ? (
         <Figure>
-          <Cover src={image.url} />
+          <ResponsiveImage
+            maxWidth="100%"
+            src={image.cropped.src}
+            ratio={image.cropped.height / image.cropped.width}
```

Comment:
Cropped annoyingly also doesn't expose the ratio (yet) —obv this is simpler for cropped but still: all of these helpers and fields should basically correspond to the props api for the tags. This is a bit difficult to use correctly.

Reply:
@damassi how prevalent is the backgroundCover abuse ResponsiveImage? To migrate this into a proper `<img>` tag we'll have to either release a breaking change or export a fresh component.

Reply:
Or maybe we just support a `srcs` prop then switch based on that.

Reply:
I don't think its that prevailent, in fact I (think) its just used once, in the VR app. ResponsiveImage itself isn't used much at all.

Reply:
Ah yeah, [it could be used in the artwork grid but it's just done in a custom fashion there](https://github.com/artsy/reaction/blob/master/src/Components/Artwork/GridItem.tsx#L115)